github/glue/action.yml

```diff
@@ -0,0 +1,43 @@
+name: "Make request to GitHub Glue"
+description: "Make request to GitHub Glue"
+inputs:
+  http_method:
+    description: "The HTTP method for the request"
+    required: false
+    default: "POST"
+  json_data:
+    description: "JSON body to send in the request"
+    required: false
+    default: "{}"
+  glue_url:
+    description: "Glue API Base URL"
+    required: true
```

Comment:
We typically store this in secrets, so we might need to pass in as a secret to the action as well vs an input